src/main/java/io/cloudsoft/jmeter/JMeterSshDriver.java

```diff
@@ -33,10 +32,11 @@ public void install() {
         List<String> urls = resolver.getTargets();
         String saveAs = resolver.getFilename();
 
-        List<String> commands = ImmutableList.<String>builder()
-                .addAll(BashCommands.commandsToDownloadUrlsAs(urls, saveAs))
-                .add("tar xvfz " + saveAs)
-                .build();
+
+        List<String> commands = Collections.emptyList();
```

Comment:
Lists returned by `emptyList` are immutable and throw `UnsupportedOperationException` if you attempt to add elements. Just write `List<String> commands = new ArrayList<String>();`